sky/clouds/azure.py

```diff
@@ -249,13 +253,15 @@ def make_deploy_resources_variables(
         # This script will modify /etc/ssh/sshd_config and add a bash script
         # into .bashrc. The bash script will restart sshd if it has not been
         # restarted, identified by a file /tmp/__restarted is existing.
+        # Also, add default user to docker group.
         # pylint: disable=line-too-long
         cloud_init_setup_commands = base64.b64encode(
```

Comment:
Same as above, do we really need to encode the cloud init to base64? Is it possible to use a plain text with some careful yaml syntax, e.g. using `: |` or `: |-` to avoid unexpected new line and indents?